tools/operator/README.md

```diff
@@ -0,0 +1,107 @@
+		`anchorpeer`		  To perform anchor peer update
+		`install`				   To install a chaincode
+		`instantiate`		    To instantiate a chaincode
+		`upgrade`			   To upgrade a chaincode
+		`invoke`				  To perform invokes by sending the traffic to a fabric network
+		`query`				   To perform queries on a fabric network
+
+- `-i` is used to pass the absolute file path for network input file. It is required
+to launch/remove fabric network. Network input file can be prepared using `networkInput.md`
+in operator
+- `-k` is used to pass the absolute file path to kube config file of kubernetes cluster.
+If `-k` is specified with `path/to/kube-config.yaml`, then it will launch/remove
+fabric network on kubernetes cluster. Kube config file can be downloaded from kubernetes
+cluster. If `-k` is not specified in the command line, operator will launch fabric
+network locally using docker-compose
+
+## Examples
+#### Fabric Network
+##### On Kubernetes Cluster
+
+To launch fabric network in kubernetes cluster, need kube config file for cluster
+and network input file
+```go run main.go -i <path/to/network spec file> -k <path/to/kube config file> -a up``` or
+```go run main.go -i <path/to/network spec file> -k <path/to/kube config file>```
+To take down the launched fabric network from the above
+```go run main.go -i <path/to/network spec file> -k <path/to/kube config file> -a down```
+
+To verify if fabric network is launched successfully or not in kubernetes cluster:
+```export KUBECONFIG=<path/to/kube config file>```
+```kubectl get pods``` - To display list of pods running in kubernetes cluster
+```kubectl get services``` - To display list of services created in kubernetes cluster
+
+##### Local
+
+To launch fabric network locally using network input file
+```go run main.go -i <path/to/network spec file> -a up``` or
+```go run main.go -i <path/to/network spec file>```
+
+To take down launched fabric network locally
+```go run main.go -i <path/to/network spec file> -a down```
+
+To verify if fabric network is launched successfully or not locally:
+```docker ps -a```
+
+#### Fabric Operations
+
+- To perform any action specified in the table above(for both the local network and the network launched in the kubernetes), use the below command
+```go run main.go -i <path/to/test input file> -a <action>```
+- To upgrade a local fabric network, use the below command
+```go run main.go -i <path/to/network spec file> -a upgradeNetwork``` 
+To upgrade a fabric network launched using kubernetes, use the below command
+```go run main.go -i <path/to/network spec file> -k <path/to kube config file> -a upgradeNetwork```
+The upgradeNetwork action will takes down an existing fabric network, upgrade all peer db's, relaunch fabric network using 2.0 images, update capabilities for orderer system channel and all application channels, update policies for consortium on orderer system channel, update policies for each organization, app policies, acls on all application channels
```

Comment:
These details in line 107 should be moved up where the action is described, near line 51.

Reply:
Done

Reply:
I think we don't need to add all this in line 51 as we are just saying what it will do. And I don't think we need to specify 2.0 here because if the fabric version gets changed in the future, we have update the README.md again.